### ISSUE TITLE
Registration error

### ISSUE THREAD

**Issue by lichenghui1476529913** [HuaiChen-1994/LDLearning#2]
Hello, I am very interested in your research work, but I keep getting registration errors when downloading the RITE datasets.

**Comment by HuaiChen-1994** [comment]
> Hello, I am very interested in your research work, but I keep getting registration errors when downloading the RITE datasets.

Hello, maybe you can download DRIVE as the Alternative data. As described in RITE,  the RITE is inherited from DRIVE. 
-For the training set, the vessel reference standard is a modified version of 1st_manual from DRIVE. 
-For the test set, the vessel reference standard is 2nd_manual from DRIVE. 
-For the A/V reference standard, four types of vessels are labeled using four colors based on the vessel reference standard. 
Since our VPN can not work these days, I am unable to confirm whether DRIVE/RITE can be correctly downloaded. 
If there still have problems, you can contact me through email.
Best wishes!
